pkg/execute/executor.go

```diff
@@ -377,32 +374,24 @@ func (e *DefaultExecutor) runVersionCommand(args []string, clusterName string) s
 	return e.findBotKubeVersion()
 }
 
-func (e *DefaultExecutor) getSortedEnabledCommands(header string, commands map[string]bool) string {
-	var keys []string
-	for key := range commands {
-		if !commands[key] {
+func (e *DefaultExecutor) getEnabledKubectlConfigs() (string, error) {
+	type kubectlCollection map[string]config.Kubectl
+
+	enabledKubectls := kubectlCollection{}
+	for name, item := range e.cfg.Executors {
+		if !item.Kubectl.Enabled {
 			continue
 		}
 
-		keys = append(keys, key)
+		enabledKubectls[name] = item.Kubectl
 	}
-	sort.Strings(keys)
 
-	if len(keys) == 0 {
-		return fmt.Sprintf("%s: []", header)
+	out := map[string]map[string]kubectlCollection{
+		"enabled": {
+			"kubectl": enabledKubectls,
+		},
 	}
 
-	const itemSeparator = "\n  - "
-	items := strings.Join(keys, itemSeparator)
-	return fmt.Sprintf("%s:%s%s\n", header, itemSeparator, items)
-}
-
-func (e *DefaultExecutor) getNamespaceConfig() (string, error) {
-	ns := e.cfg.Executors.GetFirst().Kubectl.Namespaces
-
-	out := map[string]config.Namespaces{
-		"allowed namespaces": ns,
-	}
 	var buff strings.Builder
 	encode := yaml.NewEncoder(&buff)
 	encode.SetIndent(2)
```

Comment:
I used that previously because in default setting it is using 4 spaces, and that was not in sync with the data that was printed "manually" with:
```go
	const itemSeparator = "\n  - "
	items := strings.Join(keys, itemSeparator)
```

Now we use only `yaml.Marshal` but I still think that 2 spaces are ok and not 4 are a bit too wide in the communicator window.

Reply:
Alright, but it would be great to have consistency. But keep it as it is, we'll definitely improve configs printed later 👍